integration/js/steps/ClickContentShareVideoTestButton.js

```diff
@@ -0,0 +1,30 @@
+const AppTestStep = require('../utils/AppTestStep');
+
+class ClickContentShareVideoTestButton extends AppTestStep {
+  constructor(kiteBaseTest, sessionInfo, testType) {
+    super(kiteBaseTest, sessionInfo);
+    this.testType = testType;
+  }
+
+  static async executeStep(KiteBaseTest, sessionInfo, testType) {
+    const step = new ClickContentShareVideoTestButton(KiteBaseTest, sessionInfo, testType);
+    await step.execute(KiteBaseTest);
+  }
+
+  stepDescription() {
+    return 'Click content share video test button to turn ' + (this.testType === 'ON' ? 'on' : 'off') + ' content share video test';
+  }
+
+  async run() {
+    await this.page.clickContentShareDropButton();
+    const sleep = (milliseconds) => {
+      return new Promise(resolve => setTimeout(resolve, milliseconds))
+    };
+    await sleep(1000);
```

Comment:
Dont use this. Use TestUtils.waitAround.

Reply:
done